### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by vanyachyzh**
- [DEMO LINK](https://vanyachyzh.github.io/layout_catalog_hovers/)

### CODE REVIEW

File: src/styles/blocks/header.scss [hunk 1]
```diff
@@ -38,5 +38,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    transition: all 300ms;
```
Comment (nazarmatsevych) [inline src/styles/blocks/header.scss:42]: dont use `all` here if you change only color

File: src/styles/blocks/catalog.scss [hunk 2]
```diff
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: all 300ms;
```
Comment (nazarmatsevych) [inline src/styles/blocks/catalog.scss:21]: same

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]

File: src/styles/blocks/catalog.scss (hunk 2 above)
Comment (vanyachyzh) [reply src/styles/blocks/catalog.scss:21]: but here I change two proporties transform and background-color at the same time.

Comment (nazarmatsevych) [reply src/styles/blocks/catalog.scss:21]: transition: color 300ms, transform 300ms;

And you can move duration to variable

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]

File: src/styles/blocks/header.scss [hunk 3]
```diff
@@ -38,5 +38,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    transition: color var(--duration);
```
Comment (Bohdan-Maksymiuk) [inline src/styles/blocks/header.scss:42]: why did you use css variable, use scss instead

File: src/styles/blocks/catalog.scss [hunk 4]
```diff
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: color var(--duration), transform var(--duration);
```
Comment (Bohdan-Maksymiuk) [inline src/styles/blocks/catalog.scss:21]: why did you use css variable, use scss instead

Comment (Bohdan-Maksymiuk) [reply src/styles/blocks/catalog.scss:21]: and you changed background-color, not color

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]

**Review by Viktor-Kost — APPROVED** [review]

**Pull request closed without merging**